doc/tntcxx_getting_started.rst

```diff
@@ -0,0 +1,524 @@
+
+Connecting to Tarantool from C++
+=================================
+
+To simplify the start of your working with the Tarantool C++ connector, we will
+use the `example application <https://github.com/tarantool/tntcxx/blob/master/examples/Simple.cpp>`_
+from the connector repository. We will go step by step through the application
+code and explain what each part does.
+
+The following main topics are discussed in this manual:
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. _gs_cxx_prereq:
+
+Pre-requisites
+----------------
```

Comment:
Line longer than the text → will cause a warning in the build logs.

Reply:
Corrected.